routes/web.php

```diff
@@ -12,13 +12,15 @@
 */
   Route::get('/', function () {
       return view('welcome');
-  });
+  })->middleware('guest');
```

Comment:
It's not worth it at this point in developement, but could be in the future if we had different types of users. Like adminUser, unicornUser, specialPlusUsers... ect. It'd suck to put all of that in web.php.

I'd handle access priviledges to pages in the construct of the controller that provides that page. Say for example, down the road you will only let friends view your whole profile (pay me @, location, history with me), as oppsed to just you split cred and a button make a friend request if you're a guest. That shouldn't be handled in web.php. That should be handle with special middleware and middleware groups in construct.

But for out of the box middleware like redirectifloggedin aka "guest" thats used basically only once could go in web.php for readability's sake. Its not a messy middleware to put there, because it makes things more clear than otherwise.